Frontend/CO.CDP.OrganisationApp/Pages/Registration/OrganisationDetailsSummary.cshtml

```diff
@@ -1,72 +1,119 @@
 @page "/registration/organisation-details-summary"
+@model CO.CDP.OrganisationApp.Pages.Registration.OrganisationDetailsSummaryModel
 
 <a href="/registration/organisation-registered-address" class="govuk-back-link">
     Back
 </a>
 
+@{
+    var registrationDetails = Model.RegistrationDetailModel;
+}
 
 <main class="govuk-main-wrapper">
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-two-thirds">
-            <h1 class="govuk-heading-xl">Supplier Registration</h1>
+
 
             <fieldset class="govuk-fieldset">
                 <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
-                    <h1 class="govuk-fieldset__heading">
-                        Confirm your organisation's details
+                    <h1 class="govuk-heading-l">
+                        <span class="govuk-label--l">Overview</span>
                     </h1>
+
+                    <h2 class="govuk-heading-m">Organisation details</h2>
```

Comment:
We potentially could create a summary list partial view

Reply:
Yes, we can refactor it.  Let's have dev discussion on it.